.gitignore

```diff
@@ -26,5 +26,6 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
+Index/
```

Comment:
What's this?

Reply:
IDK, something XCode is creating. I think it started appearing with XCode9.1.

Reply:
Even though I was using the default DerivedData path, turns out I had (ages ago) specified a custom path in the "Advanced" section.